openhtf/exe/__init__.py

```diff
@@ -60,6 +61,11 @@ class TestExecutor(threads.KillableThread):
 
   daemon = True
 
+  FrameworkStatus = Enum(
```

Comment:
Change to enum34 format

FrameworkStatus = Enum('FrameworkStatus',
      ['CREATED', 'START_WAIT', 'INITIALIZING', 'EXECUTING', 'STOP_WAIT',
      'FINISHING']
  )